README.md

```diff
@@ -8,36 +8,133 @@ Please fork this repository and answer any questions on this markdown document.
 
 # Linux
 * What is the command to list the contents of a direcory, line by line and ordered by size ascending in human readable format?
+		ls -laS
+
+
 * How would you add a DNS server to a network interface in Linux?
+
+		
+		
 * If the DNS server you've just added is not reachable, how can you get any particular hostname to resolve locally? 
+
+		
```

Comment:
"netstat -nr" to determine the gateway setup then try running "arp -a" for routing tables if the normal ping to the DNS does not work